sql/slave_stats_daemon.cc

```diff
@@ -230,18 +230,16 @@ static void *handle_slave_stats_daemon(void *arg MY_ATTRIBUTE((unused))) {
 bool start_handle_slave_stats_daemon() {
   DBUG_ENTER("start_handle_slave_stats_daemon");
 
-  channel_map.rdlock();
+  channel_map.assert_some_lock();
```

Comment:
Sorry for delay. Is this working when start_handle_slave_stats_daemon() is called from register_raft_followers()?

Reply:
> Is this working when start_handle_slave_stats_daemon() is called from register_raft_followers()?

@george-reynya , is there any way I can check this with the open source build?

Reply:
Oh, is the problem that you don't have the raft plugin? Let me check what the options are.